docs/sphinx/sst/understanding_sst.rst

```diff
@@ -0,0 +1,120 @@
+Its ``backend`` slot stores a reference to the memory backend that implements the timing of the main memory. Its ``cpulink`` slot stores a reference to the 
+memory link object (either a direct link or a network-on-chip endpoint) that the controller uses to communicate with upper-level components in the memory 
+hierarchy. However, instead of using the ``cpulink`` slot, the controller can also communicate with higher memory components through the ``direct_link`` port.
+In addition, the ``backendConvertor`` slot stores a reference to a memory backend convertor object which is responsible for converting memory events coming 
+from the cache hierarchy to requests used by the memory backend.
+
+Information about the different configuration parameters, ports and subcomponent slots required for simulation can be found using the command: ``sst-info memHierarchy.MemController``
+
+MemoryBackend
+*************
+The memory backend implements the timing model of the main memory. All memory backend implementations must inherit from the base abstract class ``MemBackend`` 
+present in the file ``memBackend.h`` in the **SST-Elements** source code. Class ``MemBackend`` is an abstract class and hence cannot be directly instantiated. 
+The abstract class inherits the ``SubComponent`` class and must be loaded into the slot of the memory controller, as we have already discussed above. 
+
+All memory backend models defined in SST inherit one of the three abstract classes i.e ``SimpleMemBackend``, ``FlagMemBackend`` and ``ExtMemBackend``; All these 
+abstract classes inherit the ``MemBackend`` abstract class and require a different kind of memory backend convertor.  ``SimpleMemBackend`` requires a 
+``SimpleMemoryBackendConvertor`` specified as ``memHierarchy.simpleMemBackendConvertor``, ``FlagMemBackend`` requires a 
+``FlagMemBackendConvertor`` specified as ``memHierarchy.flagMemBackendConvertor`` and ``ExtMemBackend`` requires a ``ExtMemBackendConvertor`` specified as ``memHierarchy.extMemBackendConvertor``. 
+
+Most of the memory timing models present inside SST inherit from the ``SimpleMemBackend`` abstract class and SST provides a variety of memory backends
+to choose from e.g. ``memHierarchy.simpleMem``, ``memHierarchy.simpleDRAM``, etc. 
+
+By default the ``MemoryController`` equips the ``backend`` SubComponent slot with ``memHierarchy.simpleMem`` and ``backendConvertor`` SubComponent 
+slot with ``memHierarchy.simpleMemBackendConvertor``. Different memory backends and backend memory convertors can be specified in the configuration params for 
+``MemoryController``.
+
+The memory backend is the last level in the memory hierarchy and does not need to connect to any component further down.
+
+.. rubric:: Footnotes
+.. [1] Information adapted from: Ziqi Wang, "Understanding Memory Hierarchy Simulation in SST", https://wangziqi2013.github.io.
```

Comment:
There's only really a need for the index of the source if it's linked to something I think? So we can remove it here and just state "The information relayed on this page was adapted from: ..."

Reply:
changed to a note at the top of the section